colrv1/maxcompat.toml

```diff
@@ -0,0 +1,18 @@
+
+family = "Noto Color Emoji"
+output_file = "NotoColorEmoji-COLRv1-SVG-CBDT.ttf"
+color_format = "glyf_colr_1_and_picosvg_and_cbdt"
```

Comment:
remove this .toml file and the red_pen/emoji_u270d.svg as well

Reply:
ack